uniffi_macros/src/lib.rs

```diff
@@ -84,6 +81,9 @@ pub fn setup_scaffolding(tokens: TokenStream) -> TokenStream {
         .into()
 }
 
+/// It is important for `#[uniffi::export]` to not raise unexpected errors if it
+/// fails to parse the input as this happens very often when the proc-macro is
+/// run on an incomplete input by rust-analyzer while the developer is typing.
```

Comment:
Not really clear to me what "unexpected" error would be.

Reply:
yeah, this comment was moved and I wasn't sure either. I didn't check blame but assumed you wrote that. Maybe I should just delete that entire comment?

Reply:
Ah, I didn't check the diff carefully enough. Actually the comment makes way more sense to me when placed on `#[uniffi::constructor]`. But I agree I worded it badly. The idea behind `#[uniffi::constructor]` being `#[uniffi::constructor]` is indeed that spurious "attribute not known" / "not supported" errors are avoided when the export macro fails to parse its input and outputs it verbatim as a fallback; then I think it would be confusing for there to be an error about the constructor attribute (in addition to the parse error[s]).

Reply:
Thanks - that was all very helpful.